playground/eShopLite/AppHost/aspire-manifest.json

```diff
@@ -32,7 +32,38 @@
       }
     },
     "messaging": {
-      "type": "rabbitmq.server.v0"
+      "type": "container.v0",
+      "image": "rabbitmq:3-management",
+      "env": {
+        "RABBITMQ_DEFAULT_USER": "guest",
+        "RABBITMQ_DEFAULT_PASS": "{messaging.inputs.password}"
+      },
+      "bindings": {
+        "tcp": {
+          "scheme": "tcp",
+          "protocol": "tcp",
+          "transport": "tcp",
+          "containerPort": 5672
+        },
+        "management": {
+          "scheme": "http",
+          "protocol": "tcp",
+          "transport": "http",
+          "containerPort": 15672
+        }
```

Comment:
Why does this still show up?

Reply:
Hrm. Maybe I didn't check in the updated version. I'll fix it up